src/features/user/Profile/components/RedeemModal.tsx

```diff
@@ -100,113 +100,103 @@ export default function RedeemModal({
       }}
     >
       <Fade in={redeemModalOpen}>
-        <div className={`${styles.modal} ${styles.fixModal}`}>
-          {isCodeRedeemed && validCodeData ? (
-            // after successful redeeemed code
-            <>
-              <div
-                className={styles.codeTreeCount}
-                style={{ fontSize: '2rem' }}
-              >
-                {getFormattedNumber(i18n.language, Number(validCodeData.units))}
-                <span>
-                  {t('common:tree', {
-                    count: Number(validCodeData.units),
+        {isCodeRedeemed && validCodeData ? (
+          // after successful redeeemed code
+          <div className={`${styles.modal} ${styles.fixModal}`}>
+            <div className={styles.crossButtonDiv}>
+              <button className={styles.crossButton} onClick={closeRedeem}>
+                <CancelIcon color={styles.primaryFontColor} />
+              </button>
+            </div>
```

Comment:
This is common for both true and false state of the condition `isCodeRedeemed && validCodeData` - you could move the condition below this line and remove the repetitive code.